src/main.js

```diff
@@ -122,20 +126,19 @@ function createNewIdea() {
   var currentIdea = new Idea(userNewTitle.value, userNewBody.value);
   if (userNewTitle.value && userNewBody.value) {
   savedIdeas.push(currentIdea);
+  saveIdeaToStorage();
```

Comment:
Updates the savedIdeas array and saves this new array to localStorage, overwriting what is currently saved there.